### PR TITLE
Ruqoyat

### PR DESCRIPTION
**Pull request by adebabs-tec**
please don't merge yet, it is just setup

### CODE REVIEW

**Comment by moboluwade** [comment]
Alright now, fix that soon please @adebabs-tec

**Comment by adebabs-tec** [comment]
I've fixed the conflict, you can now merge boss.

**Review by moboluwade — APPROVED** [review]
Conflicts resolved

File: node_modules/.bin/which
[the remark this answers is not in the document]
Comment (moboluwade) [reply node_modules/.bin/which:1]: The actual issue @adebabs-tec was that, you pushed a node_module in your PR. I've addressed that already, so please pull from the repo branch before making another PR

**Pull request merged**